quickstart.Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
+
+# Exposing ports
+EXPOSE 6900
+
+# Environment variables
+ENV ARGILLA_LOCAL_AUTH_USERS_DB_FILE=/users.yml
+ENV UVICORN_PORT=6900
+
+# Install Python
+RUN apt update
+RUN apt -y install curl python3.9 python3.9-dev python3.9-distutils gcc gnupg apache2-utils
+RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+RUN python3.9 get-pip.py
+
+# Install argilla
+RUN pip install argilla[server]
+
+# Install Elasticsearch
+RUN curl -fsSL https://artifacts.elastic.co/GPG-KEY-elasticsearch | apt-key add -
+RUN echo "deb https://artifacts.elastic.co/packages/7.x/apt stable main" | tee -a /etc/apt/sources.list.d/elastic-7.x.list
+RUN apt update
+RUN apt -y install elasticsearch
```

Comment:
Will this command install also the elasticsearch deps like java virtual machine and all the stuff?

Reply:
Yes it does. This step is suggested in their [docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/deb.html#deb-repo) also.

Reply:
great!